client/src/document/ingredients/browser-compatibility-table/legend.tsx

```diff
@@ -23,6 +23,7 @@ export const LEGEND_LABELS = {
   disabled: "User must explicitly enable this feature.",
   altname: "Uses a non-standard name.",
   prefix: "Requires a vendor prefix or different name for use.",
+  more: "Has more compatibility info. Click to view.",
```

Comment:
@queengooborg that's the comment that got lost:
> Please remove the _Click to view._ call to action, since it's true for others, too and we now have the _tip_.